src/include/pbs_ifl.h

```diff
@@ -558,6 +559,18 @@ typedef struct preempt_job_info {
         char	order[PREEMPT_METHOD_HIGH + 1];
 } preempt_job_info;
 
+typedef struct svr_conn {
+	int sd;                      /* File descriptor for the open socket */
+	int secondary_sd;            /* Secondary File descriptor for the open socket */
+	int state;                   /* Connection state */
+	time_t state_change_time;    /* Connnetion state change time */
```

Comment:
I think we are not using this and `last_used_time` anywhere, and I don't think we will need it also, as this was for early version of POC where we had sharding algo

Reply:
@nithinj and @bremanandjk please confirm

Reply:
It is not used in the POC. I can see it is useful for clients which are also daemons to decide when to reconnect, something which could be introduced later.